vermouth/selectors.py

```diff
@@ -48,6 +48,12 @@ def is_protein(molecule):
         for n_idx in molecule
     )
 
+def is_water(molecule):
+    # TODO: different water models.
+    return all(
+        molecule.nodes[n_idx].get('resname') == 'HOH'
+        for n_idx in molecule
+    )
```

Comment:
Can't we make this more general and simply ask the user for a resname and then a resolution (i.e. 1 to 4 or 2 to 4). The method should be general shouldn't it? I'm thinking of material science use cases where this could be very useful and this comes with very little additional code needed.

Reply:
Possible. The grouping processor takes a selector as argument, which is used to select the molecules to group. I partially agree with setting the resolution+deviations/tolerances, since those will be tied to the available mappings. I think this also hinges on a new FF file format, where it should be possible to set (default) parameters for processors, which should be overwritable from the CLI

Reply:
> I think this also hinges on a new FF file format, where it should be possible to set (default) parameters for processors, which should be overwritable from the CLI

Just chipping it quickly: we already use force field variables to set processor values. For instance we use this to set the bond type of the elastic network and to decide if we use CoG or CoM.

Reply:
True, but the processor at the moment only need the origin system, and knows nothing about the target ff. 
How about making the martinize2 bin take the required cluster size+tolerance from the target ff, and pass that on to the processor?

Reply:
That would not shock me.